### ISSUE TITLE
Boot manager for armvu devices

### ISSUE THREAD

**Issue by angelheart150** [gutosie/neoboot#12]
Boot manager is not working with armvu like Vu+zero4k 
It is working only with octagon 4008

![FB_IMG_1703783878918.jpg](https://github.com/gutosie/neoboot/assets/67199651/a0601da0-66f5-4243-9ef9-8a257e44f0d1)

**Comment by gutosie** [comment]
Bootmanager not working with armvu like Vu+Ultimo4k

czw., 28 gru 2023 o 03:53 Mohammed Elsafty ***@***.***>
napisał(a):

**Comment by angelheart150** [comment]
> Bootmanager not working with armvu like Vu+Ultimo4k
> 
> czw., 28 gru 2023 o 03:53 Mohammed Elsafty ***@***.***>
> napisał(a):
> >
> > Boot manager not working with armvu like Vu+zero4k
> > It is working only with octagon 4008
> >
> > —
> > Reply to this email directly, view it on GitHub, or unsubscribe.
> > You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
> 

Yes can you make it in the next version for these devices by updating the codes here


![293321429-fa5588c4-c8f0-4eac-bcf3-0925aa23966f](https://github.com/gutosie/neoboot/assets/67199651/56661600-83b1-425e-a7ca-7c568d33e65c)

**Comment by gutosie** [comment]
Using the boot manager for the VU+ receiver is not recommended, the images
will not start
There will be no improvement for the VU+ receiver for the boot manager

pt., 29 gru 2023 o 08:47 Mohammed Elsafty ***@***.***>
napisał(a):
